library/spdm_common_lib/libspdm_com_context_data.c

```diff
@@ -2649,13 +2649,15 @@ libspdm_return_t libspdm_init_context_with_secured_context(void *spdm_context,
         sizeof(context->transcript.message_m.buffer);
 #endif
     context->response_state = LIBSPDM_RESPONSE_STATE_NORMAL;
-    context->local_context.version.spdm_version_count = 3;
+    context->local_context.version.spdm_version_count = 4;
     context->local_context.version.spdm_version[0] = SPDM_MESSAGE_VERSION_10 <<
```

Comment:
What does this actually do? Does this say the local endpoint supports all SPDM versions?

Reply:
Right.

Otherwise, there is no way to validate the 1.3 support.